service/oracle/service.go

```diff
@@ -0,0 +1,87 @@
+package oracle
```

Comment:
I'm not sure if this is the right way.
But my intention is to merge similar functions.

It will be used to `get-oracle-key` CMD

Reply:
It has been removed.